docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,280 @@
+| [`excludedFunctions`](#ignore-specific-functions)                                   | A space-separated list of function names that should be ignored.                                                                   |                                                               
+| `excludedMacros`                                                                    |                                                                                                                                    |
+| [`staticLibraries`](#include-a-static-library)                                      | Includes a static library into `.klib`.                                                                                            |
+| [`libraryPaths`](#include-a-static-library)                                         | A space-separated list of directories where the cinterop tool searches for the library to be included in `.klib`.                  |
+| `packageName`                                                                       | Package prefix for the generated Kotlin API.                                                                                       |
+| [`headerFilter`](#filter-headers-by-globs)                                          | Filter headers by globs and include only them when importing a library.                                                            |
+| [`excludeFilter`](#exclude-headers)                                                 | Exclude specific headers when importing a library. Has priority over `headerFilter`.                                               |
+| [`strictEnums`](#configure-enums-generation)                                        | A space-separated list of enums that should be generated as a [Kotlin enum](enum-classes.md).                                      |
+| [`nonStrictEnums`](#configure-enums-generation)                                     | A space-separated list of enums that should be generated as integral values.                                                       |
+| [`noStringConversion`](#set-up-string-conversion)                                   | A space-separated lists of functions whose `const char*` parameters should not be auto-converted to Kotlin `String`s.              |
+| `depends`                                                                           |                                                                                                                                    |
+| `exportForwardDeclarations`                                                         |                                                                                                                                    |
+| `allowedOverloadsForCFunctions`                                                     |                                                                                                                                    |
```

Comment:
By default, C functions are considered unique by name, and between multiple with the same name only one is picked. However, user can disable this behavior by specifying these functions in `allowedOverloadsForCFunctions `